src/vse_sync_pp/analyzers/analyzer.py

```diff
@@ -265,3 +270,177 @@ def explain(self, data):
             'duration': data.iloc[-1].timestamp - data.iloc[0].timestamp,
             'terror': self._statistics(data.terror, 'ns'),
         }
+
+
+class WanderAnalyzer(Analyzer):
+    """Analyze Wander
+
+    Derived classes calculate specific Wander metric
+    """
+    locked = frozenset()
+
+    def __init__(self, config):
+        super().__init__(config)
+        # samples in the initial transient period are ignored
+        self._transient = config.parameter('transient-period/s')
+        # minimum test duration for a valid test
+        self._duration = config.parameter('min-test-duration/s')
+        # limit to 10k taus for now, covers most masks
+        taus_a = np.array([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 20, 30, 40, 50, 60, 70, 80, 90,
+                           100, 200, 300, 400, 500, 600, 700, 800, 900, 1000, 2000, 3000,
+                           4000, 5000, 6000, 7000, 8000, 9000])
+        taus_b = np.arange(10000, 100001, 5000)
+        # predefine limit observation window intervals for which to compute the statistic
+        self._taus_list = np.concatenate((taus_a, taus_b))
+        # tau-list values for which the statistic will be actually computed
+        self._taus = None
+        self._samples = None
+        self._errs = None
+        self._ns = None
+        self._rate = None
+        self._lpf_signal = None
+
+    def prepare(self, rows):
+        idx = 0
+        try:
+            tstart = rows[0].timestamp + self._transient
+        except IndexError:
+            pass
+        else:
+            while idx < len(rows):
+                if tstart <= rows[idx].timestamp:
+                    break
+                idx += 1
+        return super().prepare(rows[idx:])
+
+    def calculate_limit(self, tau):
+        # return limit based on `tau`
+        for (low, high), f in self._accuracy.items():
```

Comment:
This method is using `_accuracy` but that's not provided in the base class. Move into this class from the derived class? Or move this method down into the derived class?